getlantern.org/static/locale/lang-ru_RU.json

```diff
@@ -0,0 +1,62 @@
+{
+  "OPEN_INTERNET_FOR_EVERYONE": "Открытый интернет для каждого",
+  "DOWNLOAD_LANTERN": "Скачать Lantern",
+  "DOWNLOAD_LANTERN_MOBILE": "Отправить ссылку для скачивания",
+  "LANTERN_DESCRIPTION_1": "Lantern это бесплатное приложение, которое доставляет быстрый надежный и безопасный доступ к открытому интернету.",
+  "HOW_DOES_LANTERN_WORK": "Как работает Lantern?",
+  "LANTERN_DESCRIPTION_2": "Lantern это интернет-прокси, который дает вам доступ на заблокированные сайты.",
+  "LANTERN_DESCRIPTION_2": "Lantern is an Internet proxy tool that lets you access blocked sites.",
```

Comment:
Hey @just-boris, this entry `LANTERN_DESCRIPTION_2` is duplicated, could you remove it?